CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* PayPalNativeCheckout
+  * Bump native-checkout version to release `1.2.1`
```

Comment:
We should add a line to the changelog that this fixes the Google Play compliance issue so it's a bit easier to find what version of MXO resolved the issue.

Reply:
This is what we put in the release notes for direct integration. Do we want to copy and paste it? https://github.com/paypal/android-checkout-sdk/releases/tag/v1.2.1

Reply:
we don't need that entire paragraph, maybe we just hyper link to android-checkout-sdk's release notes for folks to go read for themselves the updates?